src/dds/rs-dds-device-proxy.cpp

```diff
@@ -366,4 +368,31 @@ std::shared_ptr< dds_sensor_proxy > dds_device_proxy::create_sensor( const std::
 }
 
 
+// Tagging converted profiles. dds_sensor_proxy::add_video/motion_stream tagged the raw profiles.
+void dds_device_proxy::tag_default_profiles( std::shared_ptr<stream_profile_interface> & profile,
```

Comment:
I think you was to pass by `const &` or by const value

Reply:
I meant to show that profile is updated and stream is not, but I realize now that the `const` qualifier refers to the `shared_ptr` object. Updated both to `const shared_ptr<> &`, the second points to a const stream type.